### PR TITLE
Instance: convert remaining qemu templates

### PR DESCRIPTION
**Pull request by tarruda**
Last batch of template conversions

### CODE REVIEW

File: lxd/instance/drivers/driver_qemu_templates.go [hunk 1]
```diff
@@ -538,190 +537,267 @@ func qemuDriveFirmwareSections(opts *qemuDriveFirmwareOpts) []cfgSection {
...
+				{key: "backend", value: "spicevmc"},
+				{key: "name", value: "usbredir"},
+			},
+		}, {
+			name: fmt.Sprintf(fmt.Sprintf(`device "qemu_spice-usb%d"`, i)),
+			entries: []cfgEntry{
+				{key: "driver", value: "usb-redir"},
+				{key: "chardev", value: chardev},
+			},
+		}}...)
```
Comment (tarruda) [inline lxd/instance/drivers/driver_qemu_templates.go:769]: Note that I've reordered chardev/device USB sections to make it possible creating in a single loop. If this is a problem I can keep the original order of sections by using 2 loops.

**Comment by tarruda** [comment]
> Can you take a look at fixing that in this PR also?

I haven't compared with the original formatting, but this PR should remove all inconsistency in config file formatting due to the way the `qemuAppendSections` function works:

- Entries in a section always have 0 lines between
- Sections always have 1 line between
- No indentation is ever created

In the unit tests you should be able to see exactly how each section will look like in the config file

**Comment by tomponline** [comment]
> > Can you take a look at fixing that in this PR also?
> 
> I haven't compared with the original formatting, but this PR should remove all inconsistency in config file formatting due to the way the `qemuAppendSections` function works:
> 
>     * Entries in a section always have 0 lines between
> 
>     * Sections always have 1 line between
> 
>     * No indentation is ever created
> 
> 
> In the unit tests you should be able to see exactly how each section will look like in the config file

I see thanks

**Comment by tomponline** [comment]
jenkins: test this please

**Review by tomponline — APPROVED** [review]

**Pull request merged**